f34-py39/Dockerfile

```diff
@@ -0,0 +1,46 @@
+# Thoth's extension to OpenShift's S2I build
+FROM registry.fedoraproject.org/f34/python3:latest
```

Comment:
shall we use :latest or a specific tag for better traceability?

Reply:
from this https://registry.fedoraproject.org/repo/f34/python3/tags/ it seems difficult to know which is a stable tag

Reply:
@harshad16 Are these builds submitted to build analyses endpoint? In such a case, we extract all the relevant information from logs (container hash, packages, ...) so traceability should be guaranteed even if we do not tag to a specific version or container hash.

Reply:
yes images are submitted for build analysised when these are used by the ci, however it is not submit when we create them.
i agree, I feel staying on latest tag is better , as it would be difficult to know which specific tag to use, as there is no details of them.